README.md

```diff
@@ -25,132 +25,116 @@ Package.onUse(function (api) {
 
-The imports are intended to keep shared mixins and variables for your project,
-or to allow your package to provide several components which your package's
-users can opt into one by one.
+```javascript
+api.addFiles('x.scss', 'client', { isImport: true });
+```
 
-Each compiled source file produces a separate CSS file.  (The
-`standard-minifiers` package merges them into one file afterwards.)
+Each compiled source file generates a separate CSS file, which is then merged into one by the `standard-minifiers` package.
 
 ### Importing
 
-You can use the regular `@import` syntax to import any Sass files: sources or
-imports.
+You can import styles from various locations, including other packages, your own app, and npm modules.
+
+- **Importing from another package**:
+
+```scss
+@use "meteor:{my-package:pretty-buttons}/buttons/styles" as buttons; // Assigns a namespace "buttons"
 
-Besides the usual way of importing files based on the relative path in the same
-package (or app), you can also import files from other packages or apps with the
-following syntax.
+.my-button {
+  @extend buttons.pretty-button; // Uses the imported class with its namespace
+}
+```
 
-Importing styles from a different package:
+or
 
 ```scss
-@import "{my-package:pretty-buttons}/buttons/_styles.scss";
+@use "meteor:{my-package:pretty-buttons}/buttons/styles" as *; // Imports everything into the global scope
 
 .my-button {
-  // use the styles imported from a package
-  @extend .pretty-button;
+  @extend .pretty-button; // No namespace required
 }
 ```
 
-Importing styles from the target app:
+- **Importing from your app**:
 
 ```scss
-@import "{}/client/styles/imports/colors.scss";
+@use "{}/client/styles/imports/colors.scss" as *;
 
 .my-nav {
-  // use a color from the app style pallete
-  background-color: @primary-branding-color;
+  background-color: @primary-branding-color; // Use a color from the app's style palette
 }
 ```
 
-This can also conveniently be used to import styles from npm modules for example:
+- **Importing from npm modules**:
+
 ```scss
-@import "{}/node_modules/module-name/stylesheet";
+@use "~module-name/stylesheet"; // Imports a module from node_modules
 ```
 
-Note that **Meteor 1.7** introduced a change so that files in `node_modules` aren't automatically compiled any more.
-This requires you to add a symlink inside the `imports` directory to the pacakge in order for compilation to work.
-E.g.
+If the target is a directory, it will search for an `index.scss`, `_index.scss`, `index.sass`, or `_index.sass` file inside that directory.
 
-```
-meteor npm install the-package
-cd imports
-ln -s ../node_modules/the-package .
-```
+### Global Include Path
 
-See the [meteor changelog](https://github.com/meteor/meteor/blob/devel/History.md) for more information.
+At the moment, there is no support for a global include path. If there is significant demand, this feature could be added in future updates.
```

Comment:
Isn't `includePaths` with `@forward` acts as global import?

Reply:
Currently, it seems necessary to support the `includePaths` option because, without it, the transition would place an even greater burden on users, who already face a significant effort in adapting their projects.  

Additionally, some level of support for the [Sass migrator](https://sass-lang.com/documentation/cli/migrator/) might still be required to facilitate the migration process.